CHANGELOG.md

```diff
@@ -1,8 +1,31 @@
-# dbt_xero v0.version.version
-This release includes the following updates.
+# dbt_xero v0.9.0 
+This release includes the following updates:
+
+## Breaking Changes (requires --full-refresh)
+- Added tracking categories by pivoting out fields and grabbing the latest tracking category records from invoice line items and journal line entries. 
```

Comment:
It would be great to give a bit of a reason as to why we're adding this change and highlight that this will result in schema changes.

```suggestion
- Added tracking categories by pivoting out fields and grabbing the latest tracking category records from invoice line items and journal line entries. This update will provide more granular reporting capabilities and will result in the following model changes:
```

Reply:
Committed.